thoth/user_api/api_v1.py

```diff
@@ -347,6 +347,19 @@ def list_runtime_environments():
     }
 
 
+def list_hardware_environments():
+    """List available hardware environments."""
+    environments = []
+    for solver_name in _OPENSHIFT.get_solver_names():
+        solver_info = GraphDatabase.parse_python_solver_name(solver_name)
+        environments.append(solver_info)
+
+    return {
+        "runtime_environments": environments,
+        "parameters": {}
+    }
```

Comment:
This is probably some copy-pasta error? :spaghetti: :)

Reply:
Thanks, yes!